charts/hub-agent/templates/deployment.yaml

```diff
@@ -22,6 +22,7 @@ spec:
           args:
             - --leader-elect=true
             - --enable-webhook={{ .Values.enableWebhook }}
+            - --enable-deny-webhook={{ .Values.enableDenyWebhook }}
```

Comment:
we can probably use a better name? How about something like safeguard or guardRail webhook?